build/azure-pipelines.release-template.yml

```diff
@@ -158,12 +161,18 @@ stages:
           - script: go run mage.go UseXBuildBinaries
             displayName: "Setup Bin"
           - task: Docker@1
-            displayName: Docker Login
+            displayName: Docker Hub Login
             inputs:
               containerRegistryType: Container Registry
-              dockerRegistryEndpoint: getporter-registry
+              dockerRegistryEndpoint: ${{parameters.registry}}
```

Comment:
In Azure DevOps I have created a couple docker registry authentication sets, so that depending on where we are publishing to, it uses the proper credentials.